lib/license.js

```diff
@@ -50,18 +94,18 @@ function fixCli(pkgPath) {
     switch (license.name) {
       case 'StrongLoop': // OBSOLETE
         file.data.license = 'SEE LICENSE IN LICENSE.md';
-        writeStrongloopLicense();
+        license = writeStrongloopLicense();
```

Comment:
These assignments to a local variable followed by a return are really confusing when reviewing the PR. At first I thought they were some sort of compiler warning avoidance like you see from `splint` but then I remembered this is JS and we aren't using any of the linters that are intelligent enough to care about unused return values.

That said, none of the alternatives that come to mind are any more appealing.